includes/wf_crm_webform_base.inc

```diff
@@ -417,6 +417,10 @@ abstract class wf_crm_webform_base {
    */
   protected function getRelationship($r_types, $cid1, $cid2) {
     $found = array();
+    $active_clause = '';
+    if ($this->settings['create_new_relationship']) {
```

Comment:
I think you should wrap this variable in `!empty()` to avoid a possible php notice if it's not defined.